src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -586,6 +583,7 @@ private static int getDistToVariableUsageInChildNode(DetailAST childNode, Detail
                     resultDist = 0;
                 }
                 break;
+            case TokenTypes.VARIABLE_DEF:
             default:
                 if (examineNode.findFirstToken(TokenTypes.SLIST) == null) {
                     resultDist++;
```

Comment:
can just remove such case? instead of attaching it to default ?

Reply:
done, removed